modules/prebid/rulesengine/config/config.go

```diff
@@ -94,6 +94,14 @@ func validateRuleSet(r *RuleSet) error {
 			r.ModelGroups[i].Weight = 100
 		}
 
+		if len(r.ModelGroups[i].Schema) > 0 && len(r.ModelGroups[i].Rules) == 0 {
+			return fmt.Errorf("ModelGroup %d has schema functions but no rules", i)
+		}
+
+		if len(r.ModelGroups[i].Schema) == 0 && len(r.ModelGroups[i].Rules) > 0 {
+			return fmt.Errorf("ModelGroup %d has no schema functions to test its rules against", i)
+		}
```

Comment:
I think it makes sense to add these checks. I'm curious if you manually tested it and if it was causing strange behavior. I manually ran tests a couple of weeks ago to verify we gracefully handled the case when schema and rules were empty and/or null.

Reply:
Discussed offline. No action needed.